website/docs/reference/configs-and-properties.md

```diff
@@ -1,5 +1,9 @@
 ---
 title: Configs, properties, what are they?
+sidebar_label: Configs and properties
+intro_text: "Clarify the difference between properties and configs in dbt: properties describe resources, while configs control how dbt builds them in the warehouse"
+description: "Clarify the difference between properties and configs in dbt: properties describe resources, while configs control how dbt builds them in the warehouse"
```

Comment:
We're switching between "configs" in the title and descriptions and "configurations" in the body. Should we include "Configurations" somewhere in the front matter? Thinking about how search will be impacted

Reply:
ive updated the title to configurations and updated the first paragraph. i think search returns a lot of results since we 'configurations' in many places so have made updates but i can imagine it'll still return a lot of results.